packages/public/server/src/module/Ui/templates/entity/unrevised.twig

```diff
@@ -21,6 +21,12 @@
  #}
 <div class="editor-design">
 {{ pageHeader('Unrevised learning resources').render() }}
+<h2>{% trans %} Support links for reviewing {% endtrans %}</h2>
+<ul>
+  {% for helpLink in helpLinks %}
+    <li><a href="{{ helpLink['url'] }}">{{ helpLink['title'] | trans }}</a></li>
```

Comment:
That won't work either since `helpLink['title']` is still dynamic

Reply:
Fixed with https://github.com/serlo/serlo.org/pull/164/commits/f38421482a664e3b94ce052ddc348c61817a0ee4